deploy/template.yaml

```diff
@@ -384,6 +382,15 @@ Resources:
       RequiresCompatibilities:
         - FARGATE
       TaskRoleArn: !GetAtt ECSTaskRole.Arn
+      Tags:
+        - Key: Name
+          Value: !Sub "${AWS::StackName}-TaskDefinition"
+        - Key: Product
+          Value: "GOV.UK sign in"
+        - Key: Environment
+          Value: !Sub "${Environment}"
+        - Key: Updated
+          Value: "true"
```

Comment:
Can we make updated the date as a string. So its easily reused

Reply:
Updating.